tests/rt.sh

```diff
@@ -194,6 +194,7 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   ECF_PORT=$(( $(id -u) + 1500 ))
   QUEUE=batch
 #  ACCNR=fv3-cpu
+  ACCNR=marine-cpu
```

Comment:
Do you want to change it back to fv3_cpu?

Reply:
Isn't any non-GFSv16 work not supposed to be using fv3_cpu?

Reply:
Actually, fv3-cpu is commented out here originally (not by Denise). Default is set in detect_machine.sh as nems.

Reply:
sorry, yes, that is what I say, we do not need to set ACCNR,

Reply:
I think I have the same issue on hera as on orion, which is that I didn't have project nems available so detect machine does not work. I will remove the accnr=marine-cpu.